OloEngine/src/OloEngine/Renderer/Debug/FrameCaptureManager.cpp

```
@@ -0,0 +1,231 @@
+
+#include <chrono>
+
+namespace OloEngine
+{
+    FrameCaptureManager& FrameCaptureManager::GetInstance()
+    {
+        static FrameCaptureManager instance;
+        return instance;
+    }
+
+    void FrameCaptureManager::CaptureNextFrame()
```

Cppcheck (2.19.0)

[style] The function 'CaptureNextFrame' is never used.

(unusedFunction)